### DISCUSSION TITLE
Is contributing with some more integrations in the Aspire.Hosting space encuraged?

### DISCUSSION THREAD

**Discussion by murh-lego** [microsoft/aspire#4017] [Q&A]
Basically the title.

I'm looking into adding support for standalone apache pulsar and pulsar manager in the project I am working at currently. 
I have something written in the pattern of `Aspire.Hosting.*` libs. 

Would contributing here be something that is encuraged, for example `Aspire.Hosting.Apache.Pulsar`?
Or is it something you wish to "keep at bay" and not make a package for every little thing out there?

I'm sure there's a bunch of stuff people would like to "just get". 
However, I'm not totally sure what's the temperature in this space 😄

**Comment by davidfowl** [comment]
It's a delicate balance. Ideally each project would own these aspire plugins but we haven't shipped a GA release as yet so that's unlikely to happen short term. I think this project is popular enough that it meets the bar for me to be first class (but do we add all of the apache projects???). I'd like to hear from @eerhardt as well.

**Reply by eerhardt** [reply]
Long term, I definitely don't think the `dotnet/aspire` repo can support "every little thing out there". We will have to support a model where Hosting extension packages can come from anywhere - just like any NuGet package.

I don't think we want to be exclusionary (yet) in the repo however. If something seems valuable, we should add it. We can always move it out later, if it shouldn't be there.

**Reply by maranmaran** [reply]
Thanks for replying folks

I sent myself out to try it 😄 if you find value here let me know, otherwise it was a nice experience for me anyhow 🎉 .

https://github.com/dotnet/aspire/pull/4146
